### DISCUSSION TITLE
Infinite Scrolling

### DISCUSSION THREAD

**Discussion by harikanagidi** [microsoft-search/pnp-modern-search#1936] [Ideas]
Modern search experiences support "infinite scrolling". Inside search results webpart once we scroll down we would like to have infinite scroll instead of pagination.

Any references or ideas to implement this is highly appreciated.

can we expect this in the near future?

**Comment by wobba** [comment]
Not the near future as technically it will require some re-architecting to make it happen. And, modern search solutions do not always have continuous scroll yet. Mobile does, web search for the All verticals do not. Media search do. So it all depends :)
